modules/geo-layers/src/tileset-2d/tileset-2d.ts

```diff
@@ -352,7 +356,7 @@ export class Tileset2D {
   }
 
   /** Returns a zoom level for a tile index */
-  getTileZoom(index: TileIndex) {
+  getTileZoom(this: void, index: TileIndex) {
```

Comment:
Nit: What do we mean by `this: void` syntax? Add a comment?

Reply:
Hmm, a linter or compiler failed at one point asking for it but I don't think we need it. It had to do with labelling that `this` isn't supposed to be used in the function if I remember correctly.